Address 5 issues including: Add examples of escaping syntax and add text about non-linguistic sorting

aphillips opened the pull request:
Addresses #3

Addresses #62


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/65.html" title="Last updated on May 10, 2022, 4:25 PM UTC (0b0a812)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/65/43f96d7...aphillips:0b0a812.html" title="Last updated on May 10, 2022, 4:25 PM UTC (0b0a812)">Diff</a>

aphillips committed:
Add examples of escaping syntaxes

(Fixes bp-i18n-specdev#3)

aphillips committed:
Merge remote-tracking branch 'refs/remotes/origin/gh-pages' into gh-pages

aphillips committed:
Add non-linguistic sort information

Fixes #62

aphillips committed:
Address #10 (disallow unpaired surrogates)

In this case, I added a Note to the existing prohibition, which was
probably unclear.

aphillips committed:
Address #13: improve description of BCP47

aphillips committed:
Address #19: add text about locale-neutral serialization. ack DWBP.

aphillips committed:
Fix one typo. Clarify the sentence about choices in sorting.

xfq commented on index.html:
```diff
@@ -1388,6 +1390,64 @@ <h3>Designing character escapes</h3>
+			</thead>
+			<tbody>
+				<tr>
+					<td>Hex NCRs</td>
```
```suggestion
					<td>Hex <abbr title="numeric character references">NCRs</abbr></td>
```

aphillips replied:
Good callouts. See if my edits in ff085fe address these.

xfq commented on index.html:
```diff
@@ -1388,6 +1390,64 @@ <h3>Designing character escapes</h3>
+					<td>C (UTF-32)</td>
+					<td>C</td>
+					<td><code>\U0001f63d</code></td>
+					<td>Fixed-width; encodes UTF-32 code units; most often used to encode supplementary characters together with  lowercase <code>u</code> escapes used for BMP characters, e.g. <kbd>\u00c0 \U001f63d \u12fe</kbd></td>
```
```suggestion
					<td>Fixed-width; encodes UTF-32 code units; most often used to encode supplementary characters together with  lowercase <code>u</code> escapes used for <abbr title="Basic Multilingual Plane">BMP</abbr> characters, e.g. <kbd>\u00c0 \U001f63d \u12fe</kbd></td>
```

aphillips replied:
See ff085fe

aphillips committed:
Address @xfq's comments; other cleanup
- Added glossary link to BMP
- Added `abbr` for NCRs
- Regularized the description of each escape syntax
- Fixed a couple of typos :-P
- Added text about preferring hex to decimal

aphillips committed:
Address @r12a's comment on #13

aphillips committed:
Address @r12a's comment on #10
- remove "note"
- add 'unpaired' to the mustard
- revise text per my comment

aphillips committed:
Address @r12a's comment on #62 (editorial nit)

aphillips committed:
Attemping to address @r12a's comments on #3

Reorganized the table.

aphillips committed:
Fixed typo for #3

Was missing a 0 in the \U0001f63d comment

aphillips committed:
Further edits for #3

aphillips committed:
Further edits of the escapes table

_This time for sure..._

r12a reviewed (approved):
\o/ great job
